Packs/CVE_2021_44228/Playbooks/playbook-CVE-2021-44228_-_Log4j_RCE.yml

```diff
@@ -10,17 +12,23 @@ description: "Critical RCE Vulnerability: log4j - CVE-2021-44228\n\nOn Dec. 9, 2
   \ related the log4j 0-day exploit known as CVE-2021-45105 that allows an attacker\
   \ with control over Thread Context Map data to cause a denial of service when a\
   \ crafted string is interpreted. This issue was fixed in Log4j 2.17.0 and 2.12.3.\n\
-  \n**Affected Version**\n\nApache Log4j 2.x <= 2.15.0-rc1\n\nThis playbook should\
-  \ be triggered manually or can be configured as a job.\nPlease create a new incident\
-  \ and choose the **CVE-2021-44228 - Log4j RCE** playbook and **Rapid Breach Response**\
-  \ incident type.\n\n**The playbook includes the following tasks:**\n\n* Collect\
-  \ related known indicators from several sources.\n* Indicators and exploitation\
-  \ patterns hunting using PAN-OS, Cortex XDR and SIEM products.\n*Search for possible\
-  \ vulnerable servers using Xpanse and Prisma Cloud.\n* Block indicators automatically or manually.\n\
-  \n**Mitigations:**\n* Apache official CVE-2021-44228 patch.\n* Unit42 recommended\
-  \ mitigations.\n* Detection Rules.\n    * Snort\n    * Suricata\n    * Sigma\n \
-  \   * Yara\n\nMore information:\n[Apache Log4j Vulnerability Is Actively Exploited\
-  \ in the Wild (CVE-2021-44228)](https://unit42.paloaltonetworks.com/apache-log4j-vulnerability-cve-2021-44228/)\n\
+  \nOn Dec 28 2021, another RCE vulnerability was published for Apache Log4j2, versions\
+  \ 2.0-beta7 through 2.17.0 (excluding security fix releases 2.3.2 and 2.12.4).\n\
+  In order to exploit this vulnerability, an attacker with permission to modify the\
+  \ logging configuration file can construct a malicious configuration using a JDBC\
+  \ Appender with a data source referencing a JNDI URI which can execute remote code.\
+  \ This issue is fixed by limiting JNDI data source names to the java protocol in\
+  \ Log4j2 versions 2.17.1, 2.12.4, and 2.3.2.\n\n**Affected Version**\n\nApache Log4j\
+  \ 2.x <= 2.15.0-rc1\n\nThis playbook should be triggered manually or can be configured\
+  \ as a job.\nPlease create a new incident and choose the **CVE-2021-44228 - Log4j\
+  \ RCE** playbook and **Rapid Breach Response** incident type.\n\n**The playbook\
+  \ includes the following tasks:**\n\n* Collect related known indicators from several\
+  \ sources.\n* Indicators and exploitation patterns hunting using PAN-OS, Cortex\
+  \ XDR and SIEM products.\n*Search for possible vulnerable servers using Xpanse.\n\
+  * Block indicators automatically or manually.\n\n**Mitigations:**\n* Apache official\
```

Comment:
Search for possible vulnerable servers using Xpanse and **Prisma Cloud.**
where is prisma part? :)